blog.html

```diff
@@ -5,6 +5,12 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="./styles/stylesheet.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
```

Comment:
To make your life easier I'd recommend only using two fonts. It's one more thing to complicate